pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -101,6 +101,10 @@ public CompletableFuture<Void> updateTopicPoliciesAsync(TopicName topicName, Top
 
     private CompletableFuture<Void> sendTopicPolicyEvent(TopicName topicName, ActionType actionType,
                                                          TopicPolicies policies) {
+        if (NamespaceService.isHeartbeatNamespace(topicName.getNamespaceObject())) {
```

Comment:
Why don't we use `NamespaceService#isSystemServiceNamespace`?

Reply:
Keep the same with line 240.